### PR TITLE
Readme file updated

### PR DESCRIPTION
**Pull request by maddy020**
Readme file has been updated

### CODE REVIEW

**Comment by alecrem** [comment]
I'm very sorry but this repo is not too easy to collaborate on if you are not a member of our community. Instructions are often unclear like this, and it would take us a shorter time to change the README ourselves than it would take us letting you know everything that needs to be changed and how, so please do not take it personally if we sometimes cannot follow up with your activity.

I see that you are students trying to collaborate on open source projects, and I think that is great and would like to encourage you! I'm just sorry that we are a rather closed community and documentation is not too good.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,14 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+First,install dependencies
+
+```bash
+npm install
+# or
```
Comment (alecrem) [inline README.md:9]: Notice that we don't have a `package-lock.json` but just a `yarn.lock`. We don't use npm — only yarn.

So please include only instructions for yarn.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,14 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+yarn install
+
+Second, run the development server:
 
 ```bash
 npm run dev
```
Comment (alecrem) [inline README.md:15]: It would be nice if you could also remove the npm instructions that were present on the original README.

**Review by alecrem — CHANGES_REQUESTED** [review]

**Comment by maddy020** [comment]
> I'm very sorry but this repo is not too easy to collaborate on if you are not a member of our community. Instructions are often unclear like this, and it would take us a shorter time to change the README ourselves than it would take us letting you know everything that needs to be changed and how, so please do not take it personally if we sometimes cannot follow up with your activity.
> 
> I see that you are students trying to collaborate on open source projects, and I think that is great and would like to encourage you! I'm just sorry that we are a rather closed community and documentation is not too good.

Ok no issue

**Review by alecrem — APPROVED** [review]
Thank you! LGTM.

**Pull request merged**